vocabs/DID-core-shape-v1.ttl

```diff
@@ -0,0 +1,125 @@
+# This shape file is hosted at  https://www.w3.org/ns/did/shacl/v1
+#
+# You can run this (using https://pypi.org/project/pyshacl/), for example, with:
+#  pyshacl -s DID-core-shape-v1.ttl -e DID-core-v1.ttl -f human shape_test.ttl
+#
+
+@prefix did:  <https://www.w3.org/ns/did/vocab/v1#> .
```

Comment:
```suggestion
@prefix did:  <https://www.w3.org/ns/did/v1#> .
@prefix sec:  <https://w3id.org/security#> .
```

Reply:
You are correct, that is wrong... I was in the middle of lots of edits and messed this one up. Fixed.